pkg/minikube/node/start.go

```diff
@@ -101,10 +101,10 @@ func Start(cc config.ClusterConfig, n config.Node, existingAddons map[string]boo
 	showVersionInfo(n.KubernetesVersion, cr)
 
 	var bs bootstrapper.Bootstrapper
-	var kubeconfig *kubeconfig.Settings
+	var kubeconfig *kubeconfigpkg.Settings
 	if apiServer {
 		// Must be written before bootstrap, otherwise health checks may flake due to stale IP
-		kubeconfig, err = setupKubeconfig(host, &cc, &n, cc.Name)
+		kubeconfig = setupKubeconfig(host, &cc, &n, cc.Name)
```

Comment:
I don't think that this can be removed as the kubeconfig is constructed inside of the `setupKubeconfig` function, see https://github.com/kubernetes/minikube/blob/da7227eb7249cc46917f794767a3bc9a452542b4/pkg/minikube/node/start.go#L242-L266